modules/message-lane/src/lib.rs

```diff
@@ -558,11 +558,16 @@ fn ensure_operational<T: Trait<I>, I: Instance>() -> Result<(), Error<T, I>> {
 
 /// Creates new inbound lane object, backed by runtime storage.
 fn inbound_lane<T: Trait<I>, I: Instance>(lane_id: LaneId) -> InboundLane<RuntimeInboundLaneStorage<T, I>> {
-	InboundLane::new(RuntimeInboundLaneStorage {
+	InboundLane::new(inbound_lane_storage::<T, I>(lane_id))
+}
+
+/// Creates new runtime inbound lane storage.
+fn inbound_lane_storage<T: Trait<I>, I: Instance>(lane_id: LaneId) -> RuntimeInboundLaneStorage<T, I> {
```

Comment:
Do we benefit much from splitting the lane initialization from the storage initialization? I thought it was fine before

Reply:
Yes - the storage is used by tests, but it isn't accessible from the lane.